tests/RecoveryCodeCollectionTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
It would help to see how compatible the code is, if you add the missing php versions to the `.travis.yml`:
```
php:
    - 7.0
    - 7.1
    - 7.2
    - 7.3
    - 7.4
```

Reply:
solved with github actions. fails  for php7.0. maybe you know the error?

Reply:
This error occure because the required package does not set the correct requirement. In the `composer.json` is a requirement of version `7.0` and in the `src/Plugin.php` is a `private const` construct and this requires php-version `7.1`. You can not solve it by yourself. I would skip the support of the old version. `7.1` or better `7.2` should be the new one, but than you should readd the property-typehints 😄